### PR TITLE
Fix the bug about the include-shadow para in repo clone

### PR DESCRIPTION
**Pull request by bliuredhat**
When do repo clone with [--include-shadow | --exclude-shadow]
parameter,the command can't work well.

JIRA:PDC-1817

### CODE REVIEW

**Comment by ycheng-aa** [comment]
It's better also repo clone related tests.

**Comment by bliuredhat** [comment]
@ycheng-aa and @simozhan ; please help to review this patch.

File: tests/repo/tests.py [hunk 1]
```diff
@@ -131,3 +131,40 @@ def test_delete(self, api):
...
+                "variant_uid": "Workstation"
+            }
+
+            for x in range(1, 30)
+        ])
+        self._setup_detail(api)
+        with self.expect_output('list.txt'):
+            self.runner.run(['content-delivery-repo', 'clone',
+                             '--release-id-from', 'rhel-7.1-updates',
+                             '--release-id-to', 'rhel-7.1'
```
Comment (ycheng-aa) [inline tests/repo/tests.py:157]: This is for --include-shadow and --exclude-shadow take effect. Could also add corresponding tests?

**Comment by simozhan** [comment]
i can't run the test successfully. can you help to check it?

File: tests/repo/tests.py (hunk 1 above)
Comment (bliuredhat) [reply tests/repo/tests.py:157]: fix in https://github.com/product-definition-center/pdc-client/pull/85/commits/d3a39a99703fdf8cf925cf8b9f85394c15e86662

**Comment by simozhan** [comment]
looks good to me

**Comment by ycheng-aa** [comment]
I think list_clone_false.json should have a more meaningful name. It means shadow is false, right? Could it be something like list_clone_shadow_false.json?
Other part looks for me.

**Comment by ycheng-aa** [comment]
Looks good.

**Pull request merged**